Rakefile

```diff
@@ -303,8 +303,7 @@ def create_md5(vm_type)
 end
 
 def bundle_learning_vm
-  make_learning_vm_dir
-  copy_ova_to_dir
```

Comment:
Were these tasks already removed from the Rakefile previously?

Reply:
Yeah, those were consolidated into one method in https://github.com/puppetlabs/education-builds/pull/830/files but I missed a few things needed to make the change actually work. I just tested this version and it worked as expected.